docs/source/openfl/components.rst

```diff
@@ -0,0 +1,54 @@
+.. # Copyright (C) 2020-2021 Intel Corporation
+.. # SPDX-License-Identifier: Apache-2.0
+
+.. _openfl_components:
+
+******
+|productName| core components
+******
+
+.. toctree::
+   :maxdepth: 2
+
+   `Spawning`_
+   `Long-living`_
+
+
+.. _openfl_spawning_components:
+
+Spawning components
+##########
+
+Aggregator
+===========
+
+The aggregator is a short-living entity, which means that its lifespan is limited by experiment execution time. 
+It orchestrates collaborators according to the FL plan and performs model updates aggregation.
+The aggregator is spawned by the Director (described below) when a new experiment is submitted.
+
+
+Collaborator
+=============
+
+Collaborator is also a short living entity, it manages training the model on local data: executes assigned tasks, converts DL framework-specific tensor objects to |productName| inner representation, and exchanges model parameters with the aggregator.
+Converting tensors is done by Framework adapter plugins. |productName| ships with Pytorch and TensorFlow 2 framework adapters, this list will be extended in the future. User is free to implement their adapter for the required DL framework enabling |productName| support for experiments using this framework. The adapter plugin interface is simple: there are two required methods to load and extract tensors from a model and optimizer. Model is loaded with relevant weights before every task and at the end of the training task, weights are extracted to be sent to the central node and aggregated.
```

Comment:
"|productName| ships with Pytorch and Tensorflow 2.x framework adapters. These framework adapters are intended to be extensible, and we encourage users to contribute new adapters for DL frameworks they would like to see supported in |productName|."

Reply:
Is the Framework Adapter documented elsewhere? The content is good here, but we should add a dedicated section to the documentation that walks through the plugin interface

Reply:
Accepted proposed changes and added a dedicated section for plugin components